lib/cli/index.js

```diff
@@ -312,9 +312,26 @@ var _ = require('lodash'),
      */
     _run = () => {
         var runParser = new argparse.ArgumentParser({
-            debug: true,
-            addHelp: false
-        });
+                debug: true,
+                addHelp: false
+            }),
+
+            /**
+             * Parses `=` separated key-value pairs into Postman variable list like arrays. `=` characters in values are
+             * also supported.
+             *
+             * @param {String} arg - The argument provided to `--global-var`.
+             * @returns {{key, value}} - The object representation of the current CLI variable.
+             * @throws {Error} - An error is thrown for arguments that are not in the form foo=bar.
+             */
+            cliVar = function (arg) {
+                var eqIndex = arg.indexOf('=');
+
+                if (eqIndex === -1) { throw new Error('CLI global vars must be specified in a `key=value` format'); }
```

Comment:
We can make `eq` optional and define the var with undefined value. Would negate the need to throw error. Later we anyway plan to augment CLI reporter with summary of environemnt and globals uses, that would aide debugging.